ui/src/kapacitor/components/TickscriptHeader.js

```diff
@@ -20,18 +23,40 @@ const TickscriptHeader = ({
         <LogsToggle
           areLogsVisible={areLogsVisible}
           areLogsEnabled={areLogsEnabled}
-          onToggleLogsVisbility={onToggleLogsVisbility}
+          onToggleLogsVisibility={onToggleLogsVisibility}
         />}
       <div className="page-header__right">
         <SourceIndicator />
-        <button
-          className="btn btn-success btn-sm"
-          title={id ? '' : 'ID your TICKscript to save'}
-          onClick={onSave}
-          disabled={!id}
-        >
-          {isNewTickscript ? 'Save New TICKscript' : 'Save TICKscript'}
-        </button>
+        {isNewTickscript
+          ? <button
+              className="btn btn-success btn-sm"
+              title="Name your TICKscript to save"
+              onClick={onSave}
+              disabled={!id}
```

Comment:
Can you help me reason about this line? Will `id` be null if it's a new tickscript?

Reply:
I believe this only applies to "new" tickscripts, as they cannot be saved without a specified `id`